app/assets/stylesheets/custom.css.scss

```diff
@@ -31,17 +39,45 @@ body {
   }
 }
 
-.footer {
-  position: absolute;
-  bottom: 0;
-  width: 100%;
-  height: 60px;
-  background-color: #f5f5f5;
-  p {
-    margin: 20px 0;
-    text-align: right;
-    font-size: 0.8em;
-  }
+.smfont {
+  font-size: 0.9em;
+}
+
+.warning {
+  color: $warning;
+  padding-left: 30px;
+}
+
+.card {
+  background-color: $white;
+  border: 1px solid transparent;
+  border-color: $blue;
+  border-radius: 4px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
+}
+
+.card_edit {
+  border-color: $dark-blue;
+}
+
+.card__heading {
+  color: $white;

```

Comment:
Properties should be ordered background-color, border-bottom, border-color, border-top-left-radius, border-top-right-radius, color, font-size, margin, padding